File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_2/exercice12.c
#include <stdio.h>
int n_centaines,n_dizaines,n_unités,n;
int main(){
    do{
        printf("saisir un entier positif inférieur à 999: ");
        scanf("%d",&n);
    }while(n>=999);
    n_centaines=n/100;
    n_dizaines=n%100/10;
    n_unités=n%100%10;
    printf("le nombre des centaines est: %d\n",n_centaines);
    printf("le nombre des dizaines est: %d\n",n_dizaines);
    printf("le nombre des unités est: %d\n",n_unités);
    return 0;
}

File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_2/exercice12.py
while True :
    n=int(input("saisir un entier positif inférieur à 999:  "))
    if(n<999):
        break
n_centaines=n//100
n_dizaines=n%100//10
n_unités=n%100%10
print("le nombre des centaines: ",n_centaines)
print("le nombre des dizaines: ",n_dizaines)
print("le nombre des unités : ",n_unités)